src/main/java/net/coderbot/iris/gl/shader/ShaderConstants.java

```diff
@@ -38,20 +36,36 @@ private Builder() {
 
         }
 
-        public void define(String name) {
-            this.define(name, EMPTY_VALUE);
+        public Builder define(String name) {
+        	if (name != null) {
```

Comment:
Instead of silently ignoring null values, it's probably better to use Objects.requireNonNull